types.go

```diff
@@ -105,3 +105,32 @@ func ParseScaleEvent(scaleEventStr string) (ScaleEvent, error) {
 		return "", errors.New(fmt.Sprintf("Unknown scale event: %s", scaleEventStr))
 	}
 }
+
+type Duration struct {
+	time.Duration
```

Comment:
why?

Reply:
time.Duration does not implement pretty print, and for example 1 hour will be printed as `3600000000000`